docs/usage/self-hosted-configuration.md

```diff
@@ -240,11 +240,12 @@ This reduces the chance of unintended consequences with global Git configs on sh
 
 ## gitUrl
 
-Override the default resolution for git remote, e.g. to switch GitLab from HTTPS to SSH-based. Currently works for GitLab only.
+Override the default resolution for Git remote, e.g. to switch GitLab from HTTPS to SSH-based.
+Currently works for GitLab only.
 
 Possible values:
 
-- `default`: use HTTP URLs provided by the platform for Git
+- `default`: use HTTPS URLs provided by the platform for Git
```

Comment:
@mbrandau I think you meant `HTTPS` here? Or does `HTTP` work as well with GitLab?

Reply:
My bad! Of course HTTPS is right.

Reply:
Thank you for confirming that HTTPS is the correct term. 😄